src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory-GetTokenAcquirers.cs

```diff
@@ -112,6 +107,14 @@ public ITokenAcquirer GetTokenAcquirer(string authenticationScheme = "")
             }
             return acquirer;
         }
+        private void CheckServiceProviderNotNull()
```

Comment:
I wonder about the performance impact. Is it an r9 rule?

Reply:
They do something else for exceptions around parameter checks, which I will do next.